src/features/page_view_timing/aggregate/index.js

```diff
@@ -50,7 +50,7 @@ export class Aggregate extends AggregateBase {
         if (!pageStartedHidden) { // see ios-version.js for detail on this following bug case; tldr: buffered flag doesn't work but getEntriesByType does
           const paintEntries = performance.getEntriesByType('paint')
           paintEntries.forEach(entry => {
-            if (entry.name === 'first-contentful-paint') {
+            if (entry?.name === 'first-contentful-paint') {
```

Comment:
This shouldn't be possible to be `null|undefined`, given `getEntriesByType` returns an empty array at minimum